<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SigMFFileSinkGUI</class>
 <widget class="RollupWidget" name="SigMFFileSinkGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>458</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>552</width>
    <height>102</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Liberation Sans</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>SigMF File Sink</string>
  </property>
  <property name="statusTip">
   <string>SigMF File Sink</string>
  </property>
  <widget class="QWidget" name="settingsContainer" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>100</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>550</width>
     <height>100</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Settings</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0">
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="deltaFrequencyLayout">
      <property name="rightMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QLabel" name="deltaFrequencyLabel">
        <property name="minimumSize">
         <size>
          <width>16</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Df</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ValueDialZ" name="deltaFrequency" native="true">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>16</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Liberation Mono</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="toolTip">
         <string>Demod shift frequency from center in Hz</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="deltaUnits">
        <property name="text">
         <string>Hz </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="decimationLabel">
        <property name="text">
         <string>Dec</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="decimationFactor">
        <property name="maximumSize">
         <size>
          <width>55</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Decimation factor</string>
        </property>
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>16</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>32</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>64</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="channelRateText">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Sink rate (kS/s)</string>
        </property>
        <property name="text">
         <string>0000k</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="recordNbTracks">
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Number of captures (tracks) in record updated at end of track</string>
        </property>
        <property name="text">
         <string>#000</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="recordTimeText">
        <property name="minimumSize">
         <size>
          <width>52</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Record time (HH:MM:SS)</string>
        </property>
        <property name="text">
         <string>00:00:00</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="recordSizeText">
        <property name="minimumSize">
         <size>
          <width>52</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Data file size (k: kB, M: MB, G: GB)</string>
        </property>
        <property name="text">
         <string>999.99M</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="decimationShiftLayout">
      <property name="rightMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QCheckBox" name="fixedPosition">
        <property name="toolTip">
         <string>Use fixed frequency shift positions for little performance improvement</string>
        </property>
        <property name="text">
         <string>Pos</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="position">
        <property name="toolTip">
         <string>Center frequency position</string>
        </property>
        <property name="maximum">
         <number>2</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="filterChainIndex">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Filter chain hash code</string>
        </property>
        <property name="text">
         <string>000</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ButtonSwitch" name="spectrumSquelch">
        <property name="toolTip">
         <string>Toggle spectrum squelch recording</string>
        </property>
        <property name="text">
         <string>SQ</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDial" name="squelchLevel">
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Spectrum squelch level (dB)</string>
        </property>
        <property name="minimum">
         <number>-120</number>
        </property>
        <property name="maximum">
         <number>0</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>-50</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="squelchLevelText">
        <property name="toolTip">
         <string>Spectrum squelch level (dB)</string>
        </property>
        <property name="text">
         <string>-100</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDial" name="preRecordTime">
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Pre-recording time (s)</string>
        </property>
        <property name="maximum">
         <number>10</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="preRecordTimeText">
        <property name="toolTip">
         <string>Squelched recoding pre-recording time (s)</string>
        </property>
        <property name="text">
         <string>10</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDial" name="postSquelchTime">
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Squelched recording post-recording time (s) </string>
        </property>
        <property name="maximum">
         <number>10</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="postSquelchTimeText">
        <property name="toolTip">
         <string>Squelched recording post-recording time (s)</string>
        </property>
        <property name="text">
         <string>10</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ButtonSwitch" name="squelchedRecording">
        <property name="toolTip">
         <string>Squelched recording enable</string>
        </property>
        <property name="text">
         <string>REC</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="fileNameLayout">
      <item>
       <widget class="ButtonSwitch" name="record">
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../sdrgui/resources/res.qrc">
          <normaloff>:/record_off.png</normaloff>:/record_off.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="showFileDialog">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Open file</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../sdrgui/resources/res.qrc">
          <normaloff>:/preset-load.png</normaloff>:/preset-load.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="fileNameText">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>File currently opened</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>541</width>
     <height>351</height>
    </rect>
   </property>
   <property name="windowTitle">
    <string>Channel Spectrum</string>
   </property>
   <layout class="QVBoxLayout" name="SpectrumLayout">
    <property name="spacing">
     <number>10</number>
    </property>
    <item>
     <widget class="GLSpectrum" name="glSpectrum" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>300</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Liberation Mono</family>
        <pointsize>8</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="GLSpectrumGUI" name="glSpectrumGUI" native="true"/>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RollupWidget</class>
   <extends>QWidget</extends>
   <header>gui/rollupwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ButtonSwitch</class>
   <extends>QToolButton</extends>
   <header>gui/buttonswitch.h</header>
  </customwidget>
  <customwidget>
   <class>ValueDialZ</class>
   <extends>QWidget</extends>
   <header>gui/valuedialz.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GLSpectrum</class>
   <extends>QWidget</extends>
   <header>gui/glspectrum.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GLSpectrumGUI</class>
   <extends>QWidget</extends>
   <header>gui/glspectrumgui.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../sdrgui/resources/res.qrc"/>
 </resources>
 <connections/>
</ui>
